Asgardia.Constitution.Proposal.txt

```diff
@@ -1,9 +1,9 @@
 ﻿CONSTITUTION
-OF THE SPACE STATE OF ASGARDIA
+OF THE SPACE NATION OF ASGARDIA

```

Comment:
As long as we are consistent, then I'm happy to use "Nation" instead of "State."